src/Standards/Generic/Docs/Commenting/DocCommentStandard.xml

```diff
@@ -0,0 +1,268 @@
+    ]]>
+    </standard>
+    <standard>
+    <![CDATA[
+    DocBlock must not be empty.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: DocBlock with some content.">
+        <![CDATA[
+/**
+ * <em>Some content.</em>
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Empty DocBlock.">
+        <![CDATA[
+/**
+ * <em></em>
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The opening and closing DocBlock tags must be the only content on the line.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: The opening and closing DocBlock tags have to be on a line by themselves.">
+        <![CDATA[
+<em>/**</em>
+ * Short description.
+ <em>*/</em>
+        ]]>
+        </code>
+        <code title="Invalid: The opening and closing DocBlock tags are not on a line by themselves.">
+        <![CDATA[
+<em>/** Short description. */</em>
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The DocBlock must have a short description, and it must be on the first line.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: DocBlock with a short description on the first line.">
+        <![CDATA[
+/**
+ * <em>Short description.</em>
+ */
+        ]]>
+        </code>
+        <code title="Invalid: DocBlock without a short description or short description not on the first line.">
+        <![CDATA[
+/**
+ * <em></em>@return int
+ */
+
+/**
+<em> *</em>
+ * Short description.
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Both the short description, as well as the long description, must start with a capital letter.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Both the short and long description start with a capital letter.">
+        <![CDATA[
+/**
+ * <em>S</em>hort description.
+ *
+ * <em>L</em>ong description.
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Neither short nor long description starts with a capital letter.">
+        <![CDATA[
+/**
+ * <em>s</em>hort description.
+ *
+ * <em>l</em>ong description.
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    There must be exactly one blank line separating the short description, the long description and tag groups.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: One blank line separating the short description, the long description and tag groups.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *<em>
+ * Long description.
+<em> *</em>
+ * @param int $foo
+ */
+        ]]>
+        </code>
+        <code title="Invalid: More than one or no blank line separating the short description, the long description and tag groups.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *
+ *
+</em>
```

Comment:
I'm not sure the blank line (without star) has value in this example, but I'm fine with leaving it as-is.